### PR TITLE
Added CombinedWritableSequenceFile to cascading2

### PR DESCRIPTION
**Pull request by dieu**

### CODE REVIEW

**Comment by dieu** [comment]
@isnotinvain / @gerashegalov can you look on this?

**Comment by isnotinvain** [comment]
LGTM!

File: cascading2/src/main/java/com/twitter/elephantbird/cascading2/scheme/CombinedSequenceFile.java [hunk 1]
```diff
@@ -56,13 +56,7 @@ private void updateJobConfForLocalSettings(JobConf conf) {
...
   }
 
-  @Override
-  public void sourceConfInit(
-      FlowProcess<JobConf> flowProcess,
-      Tap<JobConf, RecordReader, OutputCollector> tap,
-      JobConf conf ) {
-    super.sourceConfInit(flowProcess, tap, conf);
-
+  public static void sourceConfInit(JobConf conf) {
```
Comment (gerashegalov) [inline cascading2/src/main/java/com/twitter/elephantbird/cascading2/scheme/CombinedSequenceFile.java]: Overall, LGTM. Using the same name for static and instance methods is unfortunate though.

Comment (isnotinvain) [inline cascading2/src/main/java/com/twitter/elephantbird/cascading2/scheme/CombinedSequenceFile.java]: that's a good point. Maybe we can follow up with new names.

**Pull request merged**